compileAndBuildVLCKit.sh

```diff
@@ -356,7 +356,6 @@ buildLibVLC() {
         --disable-gnutls \
         --disable-lua \
         --disable-luac \
-        --disable-protobuf \
```

Comment:
Does this compile for all platforms, notably including tvOS?

Reply:
Tested with tvOS, nobody is complaining!